util.c

```diff
@@ -732,8 +735,9 @@ int insert_image(struct image *image, struct image *sub,
 	}
 
 fill:
-	image_debug(image, "adding %llu nul bytes at offset %llu\n", size, offset);
-	ret = write_zeroes(fd, size, offset);
+	image_debug(image, "adding %llu %#hhx bytes at offset %llu\n",
+			size, fillpattern, offset);
+	ret = write_zeroes(fd, size, offset, fillpattern);
```

Comment:
Maybe rename to 'write_bytes' and the parameter to 'byte' which is not as verbose as fillpattern.

Reply:
True. I used `byte` for `insert_image()` as well to keep it consistent.